src/qibocal/protocols/characterization/two_qubit_interaction/cz_virtualz.py

```diff
@@ -0,0 +1,416 @@
+                pulse.duration = flux_sequence[setup].duration
+                sequence.add(pulse)
+
+    return sequence, measure_target, virtual_z_phase, theta_pulse
+
+
+def _acquisition(
+    params: CZVirtualZParameters,
+    platform: Platform,
+    qubits: Qubits,
+) -> CZVirtualZData:
+    r"""
+    Acquisition for CZVirtualZ.
+
+    Check the two-qubit landscape created by a flux pulse of a given duration
+    and amplitude.
+    The system is initialized with a Y90 pulse on the low frequency qubit and either
+    an Id or an X gate on the high frequency qubit. Then the flux pulse is applied to
+    the high frequency qubit in order to perform a two-qubit interaction. The Id/X gate
+    is undone in the high frequency qubit and a theta90 pulse is applied to the low
+    frequency qubit before measurement. That is, a pi-half pulse around the relative phase
+    parametereized by the angle theta.
+    Measurements on the low frequency qubit yield the the 2Q-phase of the gate and the
+    remnant single qubit Z phase aquired during the execution to be corrected.
+    Population of the high frequency qubit yield the leakage to the non-computational states
+    during the execution of the flux pulse.
+    """
+    if not isinstance(list(qubits.keys())[0], tuple):
+        raise ValueError("You need to specify a list of pairs.")
+
+    # create a DataUnits object to store the results,
+    data = CZVirtualZData()
+
+    for pair in qubits:
+        # order the qubits so that the low frequency one is the first
+        ord_pair = order_pairs(pair, platform.qubits)
+
+        for target_q, control_q in (
+            (ord_pair[0], ord_pair[1]),
+            (ord_pair[1], ord_pair[0]),
+        ):
+            for setup in ("I", "X"):
+                (
+                    sequence,
+                    measure_target,
+                    virtual_z_phase,
+                    theta_pulse,
+                ) = create_sequence(
+                    platform, setup, target_q, control_q, ord_pair, params.parking
+                )
+
+                thetas = (
+                    np.arange(params.theta_start, params.theta_end, params.theta_step)
+                    + virtual_z_phase[target_q]
+                )
+                sweeper = Sweeper(
+                    Parameter.relative_phase,
+                    thetas,
+                    pulses=[theta_pulse],
+                    type=SweeperType.ABSOLUTE,
+                )
+                results = platform.sweep(
+                    sequence,
+                    ExecutionParameters(
+                        nshots=params.nshots,
+                        acquisition_type=AcquisitionType.DISCRIMINATION,
+                        averaging_mode=AveragingMode.CYCLIC,
+                    ),
+                    sweeper,
+                )
+
+                result = results[measure_target.serial]
+                prob = result.statistical_frequency
```

Comment:
```suggestion
                result = results[measure_target.serial].serialize
                prob = result["0"]
```
This should save probabiliies not as complex numbers